amy/workshops/views.py

```diff
@@ -947,36 +945,19 @@ class AllEvents(OnlyForAdminsMixin, AMYListView):
 @admin_required
 def event_details(request, slug):
     """List details of a particular event."""
-    try:
-        task_prefetch = Prefetch(
-            "task_set",
-            to_attr="contacts",
-            queryset=Task.objects.select_related("person")
-            .filter(
-                # we only want hosts, organizers and instructors
-                Q(role__name="host")
-                | Q(role__name="organizer")
-                | Q(role__name="instructor")
-            )
-            .filter(person__may_contact=True)
-            .exclude(Q(person__email="") | Q(person__email=None)),
-        )
```

Comment:
Can you tell me why this prefetch is removed?
I thought I saw `contacts` being defined somewhere else instead, but now I can't find it. Or do we just not use this?